### PR TITLE
Rails 7.2 compatibility

### PR DESCRIPTION
**Pull request by theodorton**
- [x] Wrote [good commit messages][1].
- [x] Feature branch is up-to-date with `master` (if not - rebase it).
- [x] Squashed related commits together.
- [ ] Added tests. - Not relevant; refactoring to support Rails 7.2
- [ ] Added an entry to the [Changelog](../blob/master/CHANGELOG.md) if the new 
  code introduces user-observable changes. - Not relevant; refactoring to support Rails 7.2
- [x] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences.

[1]: http://chris.beams.io/posts/git-commit/

### CODE REVIEW

**Comment by fatkodima** [comment]
Can you please squash your commits into 1?

That CI failure regarding coverage - these coverage metrics are pretty pointless imo, I would prefer to just remove it from the project.

**Comment by theodorton** [comment]
I've fixed the compatibility spec, added the new appraisal to the CI matrix and squashed.

**Comment by arikarim** [comment]
any update on this please?

**Comment by theodorton** [comment]
Updated appraisal to use the official Rails 7.2 release. I can't do much about the coverage metrics that @fatkodima already mentioned.

@jaredbeck Is there any more work needed from my side to get this to a mergeable state?

**Comment by veganstraightedge** [comment]
Hi paper trail gem folks 👋🏻 i'm adding my voice to the chorus of encouragement to merging this PR or the other one https://github.com/paper-trail-gem/paper_trail/pull/1485

**Comment by kriom** [comment]
Likewise, hello people from paper trail gem, I also add my voice and encouragement to merge this PR :heart_hands:

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -59,11 +59,14 @@ jobs:
       # have set this up with each database as a separate job, but then we'd be
       # duplicating the matrix configuration three times.
       matrix:
-        gemfile: [ 'rails_6.1', 'rails_7.0', 'rails_7.1' ]
+        gemfile: [ 'rails_6.1', 'rails_7.0', 'rails_7.1', 'rails_7.2' ]
 
         # To keep matrix size down, only test highest and lowest rubies.
         # See "Lowest supported ruby version" in CONTRIBUTING.md
         ruby: [ '3.0', '3.2' ]
```
Comment (joshRpowell) [inline .github/workflows/test.yml:66]: should this be updated to 
```yml
ruby : [ '3.1', '3.3']
```
see https://endoflife.date/ruby

**Comment by gurgelrenan** [comment]
Let's focus on #1485 . I will close this P.R

**Pull request closed without merging**